enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/StoreCopyClient.java

```diff
@@ -110,20 +110,24 @@ private void persistentCall( CatchUpRequest request, CatchUpResponseAdaptor<Stor
                 log.info( format( "Sending request '%s' to '%s'", request, from ) );
                 StoreCopyFinishedResponse response = catchUpClient.makeBlockingRequest( from, request, copyHandler );
                 successful = successfulFileDownload( response );
```

Comment:
You already log in successfulFileDownload. Can't you update the logging logic there?